release-please-config.json

```diff
@@ -83,6 +83,10 @@
     "plugins/source/azuredevops": {
       "component": "plugins-source-azuredevops"
     },
+    "plugins/source/bitbucket": {
+      "component": "plugins-source-bitbucket"
+      "release-type": "maven"
```

Comment:
Since we only publish a Docker container I don't think we need this.
You can use `simple` as in the SDK https://github.com/cloudquery/plugin-sdk-java/blob/3f5f47fed98e79d7b3a995046b9677ca356a60fb/.github/workflows/release_pr.yml#L14 and if you want to update the version in the `build.gradle` see https://github.com/cloudquery/cloudquery/blob/9dfb9aceef98fb3392b3274e4ffd22a9f3b7748e/release-please-config.json#L173C23-L173C23 and https://github.com/cloudquery/cloudquery/blob/9dfb9aceef98fb3392b3274e4ffd22a9f3b7748e/plugins/source/typeform/plugin/plugin.py#L14